Specs/Data/Cesium3DTiles/Tilesets/TilesetWithExtras/tileset.json

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
I don't think it's necessary to create a new tileset `TilesetWithExtras` just for this change. Just add `extras` properties to `Tileset`. We did a similar thing when adding the `tilesetVersion` property.

After that, make sure to keep `SampleData/Cesium3DTiles/Tilesets/Tileset/tileset.json` and `Specs/Data/Cesium3DTiles/Tilesets/Tileset/tileset.json` in sync.

`key`/`value` is maybe too generic? Maybe `name` or `id` instead? Use different values/types for the tileset-level extras and the child extras. This will help make the test more robust.

Open a PR with any modifications to 3D Tiles sample data to https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/tree/master/samples-generator. Then regenerate `tileset.json` and update here. There are formatting inconsistencies in this hand-edited file.